packages/in_app_purchase/lib/src/in_app_purchase_connection/in_app_purchase_connection.dart

```diff
@@ -16,6 +128,18 @@ abstract class InAppPurchaseConnection {
   /// Query product details list that match the given set of identifiers.
   Future<ProductDetailsResponse> queryProductDetails(Set<String> identifiers);
 
+  /// Query all the past purchases.
+  ///
+  /// The `applicationUserName` is used for iOS only and it is optional. It does not have any effects on Android.
+  /// It is the `applicationUsername` you used to create payments.
+  /// If you did not use a `applicationUserName` when creating payments, you can ignore this parameter.
+  Future<QueryPastPurchaseResponse> queryPastPurchases(
+      {String applicationUserName});
+
+  /// Get a refreshed purchase verification data.
```

Comment:
This sort of rephrases the method name. Maybe mention that this is a utility in case there's an issue with getting the verification data originally, and as long as that's not null this doesn't need to be called?